### ISSUE TITLE
Screen Reader "Deceased" Pronounced Incorrectly

### ISSUE THREAD

**Issue by fongcindy** [GSA/px-benefit-finder#933]
### What's expected

Screen reader should be pronouncing the step indicator "About the deceased"
![image](https://github.com/GSA/px-benefit-finder/assets/128498116/3a091b30-ada5-46d7-8306-274e672b6815)

### What happened

Screen reader reads "de-not complete-ceased"

### How to replicate

Use screen reader to read the page after clicking start.

**Comment by scottqueen-bixal** [comment]
found this when viewing without JS

![Image](https://github.com/GSA/px-benefit-finder/assets/37077057/93fb096a-96db-4d85-b9b1-4a6652e21e12)
